sema/query/__main__.py

```
@@ -252,23 +241,20 @@ def getdelimiter(args: argparse.Namespace):
         return ","
 
 
-def main(sysargs=None):
+def _main(*cli_args):
```

Ruff

Missing return type annotation for private function `_main`

Add return type annotation: `None`

(ANN202)